ClLike/cl_like/bacco.py

```diff
@@ -117,22 +121,25 @@ def update_pk(self, cosmo, bcmpar=None, **kwargs):
         cospar = {
             'omega_cold': cosmo['Omega_c'] + cosmo['Omega_b'],
             'omega_baryon': cosmo['Omega_b'],
-            'sigma8_cold': cosmo.sigma8(),
             'ns': cosmo['n_s'],
             'hubble': h,
             'neutrino_mass': np.sum(cosmo['m_nu']),
             'w0': cosmo['w0'],
             'wa': cosmo['wa']}
+        if np.isnan(cosmo['A_s']):
+            cospar['sigma8_cold'] = cosmo.sigma8() # in this case sigma8 is being varied, here it is interpreted as sigma8_cold
```

Comment:
I'm a bit scared of this. Isn't there some hidden thing in CCL where it uses sigma8 that might lead to inconsistencies? @damonge